src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,35 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_USER_AGE = 18;
+    private static final int MAX_USER_AGE = 123;
```

Comment:
let's remove this constant and check related to it, who knows maybe there will be an older person in the future)